### PR TITLE
Prevent race where devSettings property is called before bridge is do…

### PR DESCRIPTION
**Pull request by tom-un**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and Microsoft/react-native :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into Microsoft/react-native :thumbsup:
- [X] I am making a fix / change for the macOS implementation of react-native
- [ ] I am making a change required for Microsoft usage of react-native

#### Description of changes

The devSettings property can cause a second bridge to be created if called during the initialization of the "real" bridge.   Test that the RCTDevSettings modules has already been initialized before fetching it.

/azp run

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/Microsoft/react-native/pull/35)

### CODE REVIEW

File: React/Modules/RCTDevSettings.mm [hunk 1]
```diff
@@ -508,7 +508,12 @@ @implementation RCTBridge (RCTDevSettings)
 - (RCTDevSettings *)devSettings
 {
 #if RCT_DEV
-  return [self moduleForClass:[RCTDevSettings class]];
+  RCTDevSettings *devSettings = nil; // [TODO(OSS Candidate ISS#2710739)
+  Class klass = [RCTDevSettings class];
```
Comment (HeyImChris) [inline React/Modules/RCTDevSettings.mm]: "devSettingsClass" probably seems better?

Comment (tom-un) [reply React/Modules/RCTDevSettings.mm]: There are other instances in react-native using `klass`.  I not all of them made be me :)

**Pull request merged**